spotrec.py

```diff
@@ -409,8 +419,20 @@ class FFmpeg:
     instances = []
 
     def record(self, filename, metadata_for_file={}):
+        global _pl_track_counter
+
         self.pulse_input = _pa_recording_sink_name + ".monitor"
 
+        if _use_pl_track_counter:
+            # expect incoming file name format like '05 - artist - song' 
```

Comment:
This may lead to errors since users can change the filename format with the --filename-pattern option